### PR TITLE
Replaced 2017 sponsors section with 2018's

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1118

### CODE REVIEW

**Comment by gideonthomas** [comment]
@mmmavis (commenting only coz this PR doesn't say WIP) can we have the Presenting festival partner first?

**Comment by mmmavis** [comment]
Sure, I was referring to what's in `Mozilla Festival 2018 - Festival Guide Draft 1 & 2` that's why. I'll update my PR.

**Comment by gideonthomas** [comment]
lol thanks for catching that...the guide still needs review, but that should be the case in the guide as well.

File: components/sponsors.jsx [hunk 1]
```diff
@@ -2,62 +2,54 @@ var React = require('react');
...
-    }
-  ],
-  "Science Fair": [
-    {
-      name: `Fetch`,
-      logo: `/assets/images/team/sponsor/Fetch.png`
+      name: `Sticker Mule`,
+      logo: `/assets/images/team/sponsor/2018/StickerMule.svg`,
+      logoClass: `sticker-mule`,
+      link: `https://www.stickermule.com/uses/hexagon-stickers?utm_source=MozFest-2018&utm_campaign=MozFest-2018-sponsorship&utm_medium=referral`
```
Comment (gideonthomas) [inline components/sponsors.jsx]: this link should be https://mule.to/mozfest-2018

**Pull request merged**